### ISSUE TITLE
Make content submission companion app

### ISSUE THREAD

**Issue by j-hannah** [j-hannah/js-report#59]
As @sirkitree mentioned, it would be cool to have an app that allowed the user to provide a link and have the fields auto-populated and then posted to Contentful. There is so much cool and useful stuff in a little app like this.

**Relevant links:**
https://auth0.com/
https://firebase.google.com/docs/auth/
https://www.contentful.com/developers/docs/concepts/apis/#content-management-api

**First ideas for requirements**
1. Needs authentication.
2. First form accepts a link
3. Link is fetched and data processed
4. Second form is populated with processed values for review prior to submission
5. Editorial rule enforcement?
6. Submission to Contenful API. 
7. Send notification to admins

**Comment by sirkitree** [comment]
3 and 4 could be done all on the same screen even. Use markdown preview, and something to pull og tags. ex: https://developers.facebook.com/tools/debug/sharing/?q=lullabot.com

**Comment by j-hannah** [comment]
@sirkitree Yeah, sounds good. I'm almost done writing a package for the meta tag scraping. I'm sure there are already 10 other packages already available, but I still wanted to write it myself just for the heck of it. Maybe publish to npm.

**Comment by j-hannah** [comment]
Here's the package I mentioned above:  

https://www.npmjs.com/package/meta-scraper

It will grab the meta and we can then use it to pre-populate the form.

**Comment by sirkitree** [comment]
Nice!
